cmd/service.go

```diff
@@ -34,7 +34,7 @@ var serviceCreateCmd = &cobra.Command{
 
 If service name is not provided, service type value will be used.
 
-A full list of service types that can be deployed is available using: 'odo service catalog'`,
+A full list of service types that can be deployed are available using: 'odo catalog list services'`,
 	Example: `  # Create new mysql-persistent service from service catalog.
   odo service create mysql-persistent
```

Comment:
To be consistent, the syntax of this command should be 

- creation : `odo catalog service create <name_of_the_service_class>`
- deletion : `odo catalog service delete mysql-persistent`

WDYT ?

Reply:
`odo catalog`  is supposed to only list and search images and services
while `odo create` or `odo delete` will deal with components and
`odo service xxx` will deal with services